lua/bufferline/options.lua

```diff
@@ -42,6 +42,8 @@ local DEFAULT_DIAGNOSTICS = {
 --- @field inactive? boolean
 --- @field visible? boolean
 
+--- @alias bufferline.options.icons boolean|"both"|"buffer_number_with_icon"|"buffer_numbers"|"numbers"
```

Comment:
The return type for `options.icons` was only set to `boolean`, when that wasn't actually the complete set of valid inputs.